lib/Molmed/Sisyphus/Common.pm

```diff
@@ -934,18 +934,15 @@ sub getBcl2FastqVersion{
     if(defined $self->{BCL2FASTQVERSION}){
 	return $self->{BCL2FASTQVERSION};
     }
-
-    my $file = "$rfPath/bcl2fastq.version";
-    if(-e $file){
-	open FILE, $file or die "Failed to open $file!\n";
-	$self->{BCL2FASTQVERSION} = <FILE>;
-	close(FILE);
-    }elsif(-e "$file.gz"){
-	open(FILE, '<:gzip', "$file.gz") || confess "Failed to open $file.gz\n";
-	$self->{BCL2FASTQVERSION} = <FILE>;
-	close(FILE);
-    }
     
+    open my $bcl2fastqfile, '<', "$rfPath/bcl2fastq_version";
```

Comment:
It would be nice if the case of not being able to open the file was handled here.